src/components/modals/FeedbackModal.jsx

```diff
@@ -44,6 +56,14 @@ export const FeedbackModal = ({ visible, setVisible }) => {
         setVisible={setFeedbackEditModalVisible}
         review={editedReview}
       />
+      <DeleteModal
+        title={t('Confirm the deletion')}
```

Comment:
All text should be container into src/localization/strings/[en.js and uk.js], please add text to translations alphabetically